src/components/common/SelectPhotos.tsx

```diff
@@ -51,8 +51,8 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 
 type SelectPhotosProps = {
-  title: string;
-  description: string | string[];
+  title?: string;
+  description?: string | string[];
```

Comment:
do conditional rendering below? otherwise we will be using up some space/margin, etc...
```
{title ? (...) : null}
{description ? (....) : null}
```